cmd/bnsd/app/examples.go

```diff
@@ -41,43 +42,53 @@ func makePrivKey(seed string) *crypto.PrivateKey {
 
 // Examples generates some example structs to dump out with testgen
 func Examples() []commands.Example {
+	metadata := &weave.Metadata{Schema: 1}
+
 	wallet := &namecoin.Wallet{
-		Name: "example",
+		Metadata: metadata,
+		Name:     "example",
 		Coins: []*coin.Coin{
 			&coin.Coin{Whole: 50000, Ticker: "ETH"},
 			&coin.Coin{Whole: 150, Fractional: 567000, Ticker: "BTC"},
 		},
 	}
 
+	eth := &coin.Coin{Whole: 50000, Fractional: 50000, Ticker: "ETH"}
```

Comment:
Can we use different values for `Whole` and `Fractional`, just to be sure …

Reply:
Done